src/main/java/world/bentobox/bentobox/managers/BlueprintsManager.java

```diff
@@ -1,43 +1,96 @@
 package world.bentobox.bentobox.managers;
 
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.lang.reflect.ParameterizedType;
+import java.lang.reflect.Type;
+import java.util.Collections;
+import java.util.EnumMap;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Objects;
+import java.util.jar.JarFile;
+
+import org.bukkit.World;
+import org.bukkit.util.Vector;
 import org.eclipse.jdt.annotation.NonNull;
+import org.eclipse.jdt.annotation.Nullable;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.InstanceCreator;
+
+import net.md_5.bungee.api.ChatColor;
 import world.bentobox.bentobox.BentoBox;
 import world.bentobox.bentobox.api.addons.GameModeAddon;
+import world.bentobox.bentobox.api.blueprints.BP_Block;
 import world.bentobox.bentobox.api.blueprints.Blueprint;
 import world.bentobox.bentobox.api.blueprints.BlueprintBundle;
-
-import java.io.File;
-import java.io.FilenameFilter;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
-import java.util.zip.ZipFile;
+import world.bentobox.bentobox.blueprints.BPClipboard;
+import world.bentobox.bentobox.blueprints.BPPaster;
+import world.bentobox.bentobox.database.json.BentoboxTypeAdapterFactory;
+import world.bentobox.bentobox.database.objects.Island;
+import world.bentobox.bentobox.util.Util;
 
 /**
- * Handles {@link world.bentobox.bentobox.api.blueprints.Blueprint Blueprints}.
+ * Handles Blueprints
  * @since 1.5.0
- * @author Poslovitch
+ * @author Poslovitch, tastybento
  */
 public class BlueprintsManager {
 
+    private static final String BLUEPRINT_BUNDLE_SUFFIX = ".json";
```

Comment:
This can change. I just used .json for now. Maybe ".bun" or something would work.